File: src/test/java/TC/암호해독Test.java
package TC;

import org.assertj.core.api.Assertions;
import org.example.TC.암호해독;
import org.junit.jupiter.api.Test;

public class 암호해독Test {
    private final 암호해독 decoding = new 암호해독();

    @Test
    void solution() {
        String cipher = "dfjardstddetckdaccccdegk";
        int code = 4;
        String answer = "attack";
        String result = decoding.solution2(cipher, code);
        Assertions.assertThat(answer).isEqualTo(result);
    }
}

File: src/test/java/TC(K)/암호해독Test.kt
package `TC(K)`

import io.kotest.core.spec.style.StringSpec
import org.assertj.core.api.Assertions
import org.example.TCK.암호해독

class 암호해독Test : StringSpec(
    {
        "암호해독 테스트 코드" {
            val cipher:String="dfjardstddetckdaccccdegk"
            val code:Int = 4
            val result = 암호해독().solution(cipher,code)
            val answer = "attack"
            Assertions.assertThat(result).isEqualTo(answer)
        }
    }
)
